processing/src/main/java/io/druid/segment/incremental/IncrementalIndex.java

```diff
@@ -742,6 +773,11 @@ public int size()
     return numEntries.get();
   }
 
+  public long sizeInBytes()
```

Comment:
Consider naming this `getBytesInMemory()` so it's clear it's related to all the other bytes-in-memory stuff.

Reply:
done